show_options.mk

```
@@ -0,0 +1,115 @@
+show_build_options:
+	echo " PLATFORM = $(PLATFORM)"
+	echo " FIRMWARE_FORMAT = $(FIRMWARE_FORMAT)"
+	echo
+	echo Build Options:
+	echo "  BOOTMAGIC_ENABLE = $(BOOTMAGIC_ENABLE)"
+	echo "  MOUSEKEY_ENABLE = $(MOUSEKEY_ENABLE)"
+	echo "  EXTRAKEY_ENABLE = $(EXTRAKEY_ENABLE)"
+	echo "  CONSOLE_ENABLE = $(CONSOLE_ENABLE)"
+	echo "  COMMAND_ENABLE = $(COMMAND_ENABLE)"
+	echo "  SLEEP_LED_ENABLE = $(SLEEP_LED_ENABLE)"
+	echo "  NKRO_ENABLE = $(NKRO_ENABLE)"
+	echo "  BACKLIGHT_ENABLE = $(BACKLIGHT_ENABLE)"
+	echo "  RGBLIGHT_ENABLE = $(RGBLIGHT_ENABLE)"
+	echo "  MIDI_ENABLE = $(MIDI_ENABLE)"
+	echo "  UNICODE_ENABLE = $(UNICODE_ENABLE)"
+	echo "  BLUETOOTH_ENABLE = $(BLUETOOTH_ENABLE)"
+	echo "  AUDIO_ENABLE = $(AUDIO_ENABLE)"
+	echo "  FAUXCLICKY_ENABLE = $(FAUXCLICKY_ENABLE)"
+	echo "  HD44780_ENABLE = $(HD44780_ENABLE)"
+
+
+show_all_features: show_build_options
+	echo
+	echo Other Features:
+    ifneq ($(API_SYSEX_ENABLE),)
+	echo "  API_SYSEX_ENABLE = $(API_SYSEX_ENABLE)"
```

```suggestion
	echo "  API_SYSEX_ENABLE = $API_SYSEX_ENABLE"
```
Use parameter substitution here and on all `echo` lines below.